### PR TITLE
bump version to 3.0.0, update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,6 +3,54 @@ For more detailed information, please see the git log.
...
 
+v3.0.0 (November 16th 2016)
+---------------------------
+
+feature release
+- backward-incompatible changes:
+  - make robot always consider subtoolchains, even without --minimal-toolchains (but in reverse order) (#1973)
+  - clean up behaviour that was deprecated for EasyBuild v3.0 (#1978)
+  - change default config to use Lmod/Lua for modules tool/syntax, GC3Pie as job backend (#1985)
+  - the minimal required version of Lmod was bumped to 5.8
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: provide justification, so that people know what they are dealing with...

File: RELEASE_NOTES [hunk 2]
```diff
@@ -3,6 +3,54 @@ For more detailed information, please see the git log.
...
+    - see http://easybuild.readthedocs.org/en/latest/RPATH-support.html
+  - add support for --consider-archived-easyconfigs (#1972)
+    - see http://easybuild.readthedocs.org/en/latest/Archived-easyconfigs.html
+  - stabilize --new-pr and --update-pr (#1979)
+    - see http://easybuild.readthedocs.org/en/latest/Integration_with_GitHub.html
+- various other small enhancements, including:
+  - add support for 'devel' log level (#1815)
+  - make remove_file aware of --extended-dry-run + add dedicated unit test (#1932)
+  - add support for filtering out setting/updating particular environment variables from generated modules (#1943)
+    - see --filter-env-vars
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: very nice feature, thanks

File: RELEASE_NOTES [hunk 3]
```diff
@@ -3,6 +3,54 @@ For more detailed information, please see the git log.
...
+  - clean up output of EasyBuild bootstrap script & add version (#1944)
+  - improved ARM platform/CPU detection (#1953)
+  - use '0' as letter dir for funky software names that don't start with a letter (#1954)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: let's give an example, so people know this is for real...

File: RELEASE_NOTES [hunk 4]
```diff
@@ -3,6 +3,54 @@ For more detailed information, please see the git log.
...
+  - make bootstrap script aware of vsc-install for offline installation (#1955)
+  - add support for blas_family() and lapack_family() methods in Toolchain instances (#1961)
+  - make copy_file dry-run aware (#1963)
+  - reorganise test easyconfigs to match structure in easyconfigs repo (#1970)
+  - add a toolchain compiler option for enforcing IEEE-754 conformance (#1975)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: beter laat dan nooit... ;)

File: RELEASE_NOTES (hunk 1 above)
Comment (boegel) [inline RELEASE_NOTES]: see #1985, it's mostly about code cleanup actually... I'll just include a ref to the PR in which the requirement was bumped.

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos!

**Pull request merged**